docs/dev/libvirt-howto.md

```diff
@@ -2,6 +2,14 @@
 
 Launching clusters via libvirt is especially useful for operator development.
 
+## Automated Setup
+Ansible can complete all of the steps below to automate this process.
+All you need is `sudo`. Run this from the root of the repository. 
+
+```console
+$ ansible-playbook -i localhost, hack/ocp_libvirt_setup.yaml -K -c local
```

Comment:
`console` with `$` is the approach I usually use in new projects.  But for consistency with existing docs, this project only uses `console` for hunks that include output.  When the hunk is just a shell command (without output), we use `sh` and no prompt (more in #27).